mod-data-import-converter-storage-server/src/main/resources/templates/db_scripts/defaultData/default_marc_holdings_job_profile.sql

```diff
@@ -1,8 +1,8 @@
 INSERT INTO ${myuniversity}_${mymodule}.job_profiles (id, jsonb) values
 ('80898dee-449f-44dd-9c8e-37d5eb469b1d', '{
   "id": "80898dee-449f-44dd-9c8e-37d5eb469b1d",
-  "name": "Default - Create SRS MARC Holdings",
-  "description": "Load MARC Holdings to create SRS MARC Holdings",
+  "name": "Default - Create Holdings and SRS MARC Holdings",
+  "description": "Load MARC Holdings and inventory holdings to create SRS MARC Holdings and Holdings",
```

Comment:
could you remove "and inventory holdings" because from one MARC Holdings we create
* SRS MARC Holdings
* Inventory Holdings

Reply:
"Load MARC Holdings to create SRS MARC Holdings and Inventory Holdings" - this would be ok